docs/components/optimize/userguide/creating-reports.md

```diff
@@ -18,7 +18,8 @@ To create a custom report based on a key performance indicator (KPI) you’d lik
 2. Click the text box under **Select Process** and select the process you’d like to analyze.
 3. Select the type of report you’d like to use on the right side of the **Create new Report** box. As with dashboards, Optimize offers preconfigured templates such as heatmaps and tables. We’ll begin with a heatmap.
 4. Click **Create Report**.
-5. Set up and customize your report. Begin by naming your report in the text box at the top of the page, pre-filled with **New Report**.
+   ![heatmap example](./img/report-heatmap-example.png)
```

Comment:
should this screenshot actually go under 5.? or does it actually belong to 5. you just put it above the text on purpose? 😅

Reply:
I placed this here intentionally as this is what you would see immediately following a click of **Create report** -- if you feel moving this under the actual set up makes more sense, I'm happy to adjust.

Reply:
fine for me, just wanted to make sure the placement was intentional